repltalk/__init__.py

```diff
@@ -1090,24 +1058,28 @@ async def _resolve_report(self, id):
 		)
 
 	async def _get_all_posts(
-		self, order='new', search_query='', after=None
+		self, order='new', search_query=None, after=None
 	):
 		if after is None:
 			posts = await self.perform_graphql(
-				'posts',
-				Queries.get_all_posts,
-				order=order,
-				searchQuery=search_query
-			)
+			'ReplPostsFeed',
+			Queries.posts_feed,
+			options={
+				'order': order.title(),
+				'searchQuery': search_query
+			}			
+		)
 			return posts
 		else:
 			posts = await self.perform_graphql(
-				'posts',
-				Queries.get_all_posts,
-				order=order,
-				searchQuery=search_query,
-				after=after
-			)
+			'ReplPostsFeed',
+			Queries.posts_feed,
+			options={
+				'order': order.title(),
+				'searchQuery': search_query,
+				'after': after
+			}			
+		)
 			return posts
```

Comment:
Since these two are almost identical they should just be made into one statement.